scripts/sync-authconfig-rego.py

```
@@ -0,0 +1,163 @@
+    indented_rego = "\n".join(indented)
+
+    match = re.search(r"(              rego: \|\n)", content)
+    if not match:
+        print(f"  ERROR: could not find 'rego: |' in {path}", file=sys.stderr)
+        sys.exit(1)
+
+    start = match.end()
+
+    for i, line in enumerate(content[start:].split("\n")):
+        if line.strip() and (len(line) - len(line.lstrip())) <= 14:
+            end = start + sum(len(l) + 1 for l in content[start:].split("\n")[:i])
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)